### ISSUE TITLE
NVDA responds slowly in QQ chat software

### ISSUE THREAD

**Issue by cary-rowen** [nvaccess/nvda#13929]
This issue may require further investigation. The slow response of NVDA in QQ is a common problem, so NVDA users in China are deeply troubled.
I use the Event Tracker add-on to view events during navigation, and it seems that Tencent QQ is sending too many events causing NVDA's performance to degrade.
I would like to know what kind of help is available to NV Access to help us troubleshoot the issue.
### Steps to reproduce:
1. Download and install QQ: https://dldir1.qq.com/qqfile/qq/PCQQ9.6.2/QQ9.6.2.28760.exe
2. You do not need to log in, use the Tab key to navigate the login screen.

### Actual behavior:
Every time I press tab I hear speech after about a few hundred milliseconds.
### Expected behavior:
NVDA should respond quickly, just like on the Windows desktop.

### NVDA logs, crash dumps and other attachments:
[nvda.log](https://github.com/nvaccess/nvda/files/9147511/nvda.log)

#### NVDA installed/portable/running from source:
installed
#### NVDA version:
2022.2
#### Windows version:
Windows 10 21H2 (x64) build 19044.1826
#### Name and version of other software in use when reproducing the issue:
Tencent QQ
#### Other information about your system:
None
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
This is true for any version, but not for other screen readers such as ZDSR
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by seanbudd** [comment]
Could you provide a list of the events from the event tracker add-on (e.g. attach a text file)?

This is very likely a problem with the implementation of the application, as a starting point please submit feedback to the authors to let them know about the problem.

**Comment by cary-rowen** [comment]
Here is the log:
[nvda_log.txt](https://github.com/nvaccess/nvda/files/9177901/nvda_log.txt)

**Comment by cary-rowen** [comment]
@seanbudd 
Yes, I'll submit this question to the developers of the software at the same time, but they don't seem to be familiar with how screen readers work, it would be nice to get some useful information from the logs above.

**Comment by cary-rowen** [comment]
@seanbudd 
did you find anything from the logs?

**Comment by Adriani90** [comment]
@cary-rowen how is it now with NVDA 2024.1 Beta? Performance has evolved gradually since this issue has been reported, and I wonder if the UIA event limiting helps in this case as well?

**Comment by cary-rowen** [comment]
It doesn't help, I will update the issue if there is any progress, thank you.

**Comment by Adriani90** [comment]
@cary-rowen please provide an update with NVDA 2025.1 beta. Is NVDA still slow?

**Comment by cary-rowen** [comment]
Sure, nothing has changed, thanks

**Comment by cary-rowen** [comment]
The new version of QQ is built using Electron, and the version mentioned in this issue is already legacy.
I probably won't keep following this anymore; I'll close it for now.
